### PR TITLE
Add tests for pipedv1 executor package

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does**:

as title

**Why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: pkg/app/pipedv1/executor/executor_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			sig:      StopSignalType("unknown"),
+			ori:      model.StageStatus_STAGE_RUNNING,
+			got:      model.StageStatus_STAGE_RUNNING,
+			expected: model.StageStatus_STAGE_FAILURE,
+		},
+	}
+
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			tc.got = DetermineStageStatus(tc.sig, tc.ori, tc.got)
```
Comment (t-kikuc) [inline pkg/app/pipedv1/executor/executor_test.go:68]:
```suggestion
			got := DetermineStageStatus(tc.sig, tc.ori, tc.got)
```

How about removing tc.got? Is it impossible?

Comment (khanhtc1202) [reply pkg/app/pipedv1/executor/executor_test.go:68]: Sorry, that was my mistake 🙏

Comment (t-kikuc) [reply pkg/app/pipedv1/executor/executor_test.go]: thank you,
hahaha, I didn't notice `DetermineStageStatus(sig StopSignalType, ori, got model.StageStatus)` has `got` too,  I was confused.

**Review by t-kikuc — APPROVED** [review]

**Review by Warashi — APPROVED** [review]
👍🏻

**Pull request merged**